### ISSUE TITLE
jpeg sources differ

### ISSUE THREAD

**Issue by PeterSurda** [Homebrew/homebrew-core#68728]
If you look at https://github.com/Homebrew/homebrew-core/blob/master/Formula/jpeg.rb, it lists three different locations for source files. The first one, https://www.ijg.org/files/jpegsrc.v9d.tar.gz produces a different checksum than the one listed in the .rb file. The second one, https://dl.bintray.com/homebrew/mirror/jpeg-9d.tar.gz, produces a 404. Only the third one, https://fossies.org/linux/misc/jpegsrc.v9d.tar.gz, produces the checksum listed in the .rb file.

The modification date for the first one appears to be January 7th 2021, while the last one January 12th 2020. After unpacking both archives, and running a diff, the contents appear to be identical. Just the archives are different.

No need to use brew to verify, you can download the files however you want (I used wget).

As a result, trying to build jpeg fails.

# Bug report

## What you were trying to do (and why)

Build the jpeg bottle from source. Doesn't matter why.

## What happened (include command output)
<pre>
Error: SHA256 mismatch
Expected: 99cb50e48a4556bc571dadd27931955ff458aae32f68c4d9c39d624693f69c32
  Actual: 6c434a3be59f8f62425b2e3c077e785c9ce30ee5874ea1c270e843f273ba71ee
</pre>

## Step-by-step reproduction instructions (by running `brew install` commands)
<pre>brew install --build-bottle jpeg</pre>

**Comment by SMillerDev** [comment]
Could you report this to the upstream developers and ask why they reuploaded it?

**Comment by PeterSurda** [comment]
Not sure how to contact them.

**Comment by PeterSurda** [comment]
TLDR; the sources were updated, the mirrors and checksums need to be updated too.

> Some files in the package were corrupted and have been corrected.
> See following message for explanation.
> 
> Regards
> Guido Vollbeding
> Organizer Independent JPEG Group

**Comment by SMillerDev** [comment]
Could you make a pull request fixing the hash to the upstream one? If needed we should just drop the mirrors.

**Comment by PeterSurda** [comment]
ok

**Comment by PeterSurda** [comment]
fossies mirror has been updated in the meantime. PR for updating the checksum will follow shortly.
